sdv/data_processing/data_processor.py

```diff
@@ -107,6 +110,7 @@ def __init__(
         model_kwargs=None,
         table_name=None,
         locales=['en_US'],
+        id_columns_use_old_behavior=[],
```

Comment:
This should be `None` and the empty list to be assigned later in the logic such as:
```
id_columns_use_old_behavior = id_columns_use_old_behavior or []
```